src/lib/features/playground/playground-service.ts

```diff
@@ -134,7 +134,19 @@ export class PlaygroundService {
                         isEnabledInCurrentEnvironment: feature.enabled,
                         strategies: {
                             result: strategyEvaluationResult.result,
-                            data: strategyEvaluationResult.strategies,
+                            data: strategyEvaluationResult.strategies.map(
```

Comment:
personally I prefer to enhance HTTP specific things in the view model/function preparing JSON representation since service is about application logic, not HTTP representations. I'd place the view model function call in the controller since controller is the HTTP delivery mechanism

Reply:
Yeah, that makes sense. I'll see if that can be done without extensive code changes 🙋🏼